functions/.znap.uninstall

```diff
@@ -18,7 +18,7 @@ private repo=
 for repo in ~znap/$^@:t; do
   if [[ -d $repo ]]; then
     is-at-least 2.31 ${${=$( git --version )}[3]} &&
-        git -C $repo maintenance unregister --quiet
+        git -C $repo maintenance unregister $> /dev/null
```

Comment:
You appear to have a typo here:

```suggestion
        git -C $repo maintenance unregister &> /dev/null
```

Reply:
Thanks for catching that :)

Reply:
Once you have fixed that, I will merge your commit.